### PR TITLE
Add premove rotations to allow for cases with random orientation

### PR DESCRIPTION
**Pull request by mihlefeld**

### CODE REVIEW

File: SkewbSkills/skewbskillsscripts.js [hunk 1]
```diff
@@ -1358,13 +1358,39 @@ function ScramblePlusColour() {
...
+        firstRotationIndex = 6;
+        secondRotationIndex = 0;
+    }
+    var firstRotation = firstRotations[firstRotationIndex];
+    var secondRotation = secondRotations[secondRotationIndex];
+    var firstRotationInv = firstRotationsInv[firstRotationIndex];
+    var secondRotationInv =  secondRotationsInv[secondRotationIndex];
+    var rotations = firstRotation + secondRotation;
+    var backRotations = secondRotationInv + firstRotationInv;
+    scramblezumanzeigenAlg = rotations + scramblezumanzeigenAlg + backRotations;
```
Comment (AnnikaStein) [inline SkewbSkills/skewbskillsscripts.js:1392]: Can you confirm that it is necessary to also do the `backRotations`?

_My thought process:_
In terms of randomized first layer color to start with, `rotations` at the beginning would be enough. I can see why adding `backRotations` is desirable (it's a conjugate `setup with rotation (do actual scramble algorithm) invert rotation`). Currently, the image is not re-oriented to standard WCA orientation (white-top, green-left), so I understand with the `backRotations` this effectively places the solved color in random positions on the screen as well, while when holding the skewb in reality, the `backRotations` do not make the result more random than not having `backRotations` (you always hold the real puzzle in WCA orientation).

_Alternatives I considered:_
If we were to do the most efficient solution to get a random-starting-color (equal to a random position on the real skewb), l. 1388,1389,1391 and the `+ backRotations` in l. 1392 could be removed. But if we look at what the image in SkewbSkills shows, we need the `backRotations` as well, right? In the future, we could also implement a function to orient the scramble image into WCA orientation, then `+ backRotations` becomes obsolete. [The other variant, keeping `rotations` and removing `backRotations` would instead lead to always getting the same color, just in different positions in the scramble image, which is not what we want.]

_TL;DR:_
We can keep it the way it is, because there exists no function yet to translate the final image into WCA orientation. Once we have this one, `backRotations` can be removed.

**Review by AnnikaStein — COMMENTED** [review]
Thank you @mihlefeld for your contribution, much appreciated!

There is a comment for the implementation of rotations as a setup, but after testing the other options I considered, I believe we can leave it the way it is.

Do you think this would also be useful for the One-Looking Trainer?

File: SkewbSkills/skewbskillsscripts.js (hunk 1 above)
Comment (mihlefeld) [reply SkewbSkills/skewbskillsscripts.js:1392]: I haven't really used or looked at the one-look trainer, but if it too doesn't have randomized layer colors, then yes I think it would be useful there as well.

Comment (AnnikaStein) [reply SkewbSkills/skewbskillsscripts.js:1392]: It's very similar to this algorithm trainer, basically only adding few moves on top. If you want, it would be great if this patch could be added there as well to have the different training modes acting in a similar way.
